src/backend/distributed/sql/udfs/create_time_partitions/latest.sql

```diff
@@ -0,0 +1,52 @@
+CREATE OR REPLACE FUNCTION pg_catalog.create_time_partitions(
+    table_name regclass,
+    to_value timestamptz,
+    partition_interval INTERVAL,
```

Comment:
What if it the first arg? It's kind of strangely wedged in between to and from now.